src/credentials/FabricTable.h

```diff
@@ -351,14 +321,53 @@ class ConstFabricIterator
 class DLL_EXPORT FabricTable
 {
 public:
+    class DLL_EXPORT FabricTableDelegate
```

Comment:
If we nest, we should probably rename to remove the prefix, so that this does not stutter. Previously we had `FabricTableDelegate`. Now we would have `FabricTable::FabricTableDelegate` which stutters. Lets make this `FabricTable::Delegate` instead.